x-pack/plugin/sql/sql-client/src/main/java/org/elasticsearch/xpack/sql/client/JreHttpUrlConnection.java

```diff
@@ -48,8 +48,10 @@
      * error.
      */
     public static final String SQL_STATE_BAD_SERVER = "bad_server";
-    private static final String SQL_NOT_AVAILABLE_ERROR_MESSAGE = "request [" + SQL_QUERY_REST_ENDPOINT
+    private static final String SQL_NOT_AVAILABLE_ERROR_MESSAGE_PARAMETER = "request [" + SQL_QUERY_REST_ENDPOINT
             + "] contains unrecognized parameter: [mode]";
+    private static final String SQL_NOT_AVAILABLE_ERROR_MESSAGE_HTTPMETHOD = "Incorrect HTTP method for uri [" + SQL_QUERY_REST_ENDPOINT
```

Comment:
Is this case tested somewhere else?

Reply:
The special case of sql not being installed/disabled had no tests before this one.